From 2166f70d4831f52b28da38f81c7c597d2df54240 Mon Sep 17 00:00:00 2001
From: Ryan Young <ryan@youngryan.com>
Date: Sat, 28 Feb 2026 23:27:29 +0000
Subject: [PATCH] backend: Don't reset panel rotation

---
 src/backends/meta-monitor-manager.c | 27 ---------------------------
 1 file changed, 27 deletions(-)

diff --git a/src/backends/meta-monitor-manager.c b/src/backends/meta-monitor-manager.c
index 3cf1a38cad..f0fca8b0af 100644
--- a/src/backends/meta-monitor-manager.c
+++ b/src/backends/meta-monitor-manager.c
@@ -1312,34 +1312,7 @@ update_panel_orientation_managed (MetaMonitorManager *manager)
     }
   else
     {
-      MetaMonitorsConfig *current_config =
-        meta_monitor_config_manager_get_current (manager->config_manager);
-
       meta_orientation_manager_inhibit_tracking (orientation_manager);
-
-      /* Rotate back to normal transform when orientation goes unmanaged */
-      if (current_config)
-        {
-          g_autoptr (MetaMonitorsConfig) config = NULL;
-          g_autoptr (GError) error = NULL;
-
-          config =
-            meta_monitor_config_manager_create_for_orientation (manager->config_manager,
-                                                                current_config,
-                                                                MTK_MONITOR_TRANSFORM_NORMAL);
-
-          if (config)
-            {
-              if (!meta_monitor_manager_apply_monitors_config (manager,
-                                                               config,
-                                                               META_MONITORS_CONFIG_METHOD_TEMPORARY,
-                                                               &error))
-                {
-                  g_warning ("Failed to rotate monitor back to normal transform: %s",
-                             error->message);
-                }
-            }
-        }
     }
 }
 
-- 
2.51.0

